app.js

```diff
@@ -25,11 +25,12 @@ app.get('/', function get(req, res) {
 });
 
 app.get('/responses', function get(req, res) {
-  res.json(smartSurveyAPICall.getData(
+return smartSurveyAPICall.getData(
     config.apiToken,
     config.apiTokenSecret,
-    config.surveyID,
-    smartSurveyAPICall.parseResponse));
+    config.surveyID)
+  .then(results => res.json(results))
+  .catch(error => console.log(error));
```

Comment:
Thanks @oliverweighell I think that's a fair point.  It wasn't really in the scope of this PR but I'm happy to make the change in a separate PR.  What would you suggest?  do you meaning printing a helpful message on the page and then the stacktrace & error?